src/lang/en-US.ts

```diff
@@ -854,6 +854,7 @@ const DetailsPanel: ResolvedIntlConfig['messages'] = {
     'detailsPanel.dataPreview.header': `Data Preview`,
     'detailsPanel.dataPreview.failedParsingMessage': `Ran into an problem parsing data. This is a UI bug and does not mean there is an issue with your data.`,
     'detailsPanel.dataPreview.listView.header': `Key`,
+    'detailsPanel.dataPreview.hidden': `Data previews are disabled in this tenant.`,
```

Comment:
I feel like this should read: `Data previews are disabled for this tenant.`

Reply:
That's fair. I think that's normally how we speak about it.